src/coord/src/coord.rs

```diff
@@ -5037,6 +5038,7 @@ impl<S: Append + 'static> Coordinator<S> {
         Ok(result)
     }
 
+    #[tracing::instrument(level = "debug", skip(self))]
     async fn send_builtin_table_updates(&mut self, updates: Vec<BuiltinTableUpdate>) {
```

Comment:
For this, I would instrument 2 more places
`fn bootstrap`, so we know when we are sending them as part of bootstrap
and, if @mjibson thinks its a good idea, `catalog_transact`

Reply:
Yes, I'd add those two places for sure.